collectors/charts.d.plugin/speedtest/README.md

```diff
@@ -0,0 +1,76 @@
+<!--
+title: "Speedtest monitoring with Netdata"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/charts.d.plugin/speedtest/README.md
+sidebar_label: "Speedtest"
+-->
+
+# Speedtest monitoring with Netdata
+
+[Speedtest CLI](https://www.speedtest.net/apps/cli) is a Linux native command line tool used to monitor internet connection performance. 
+
+This module monitors results and metrics related to [Speedtest](https://www.speedtest.net/apps/cli).
+
+## Requirements
+
+For all nodes you are going to monitor speedtest results from:
+- Install Speedtest CLI by following the installation instructions mentioned [here](https://www.speedtest.net/apps/cli).
+- Accept the license agreement as the Netdata by running `speedtest` from the command line
+
+```bash
+sudo -u netdata speedtest
+```
+
+## Metrics
+
+All metrics have "speedtest." prefix.
+
+| Metric              |    Scope    |  Dimensions   |     Units     |
+|---------------------|:-----------:|:-------------:|:-------------:|
+| download            |   global    |  bandwidth    |  bps          |
+| upload              |   global    |  bandwidth    |  bps          |
```

Comment:
- `bandwidth` is not good, it refers to capacity (not transmitted data). Just keep dimension names as `download` and `upload`. But the metric name cannot be `download` or `upload`.
- If the value in bits: `bps` => `bits/s`.

Reply:
> bps => bits/s.

Not sure the dashboard does units conversion for bits/s. If not - converting bits/s to kilobits/s will do.